publish/package.json

```diff
@@ -27,6 +27,7 @@
     "@angular/cli": "6.0.7",
     "classlist.js": "^1.1.20150312",
     "ids-enterprise": "^4.8.0-dev.20180621",
-    "jquery": "3.3.1"
+    "jquery": "3.3.1",
+    "@types/jquery": "3.3.1"
```

Comment:
Can you move `@types/jquery` up to where the other types are included?
They are toward the top of the devDependencies

Reply:
do you mean it should be under @angular/cli to make it in alphabetical order ?